R/cluster.R

```diff
@@ -39,54 +57,81 @@
 #'
 #' # Cluster on rows using Kmeans
 #' tse <- cluster(tse, KmeansParam(centers = 3))
-#'
+#' 
+#' # Clustering done on the samples using Hclust
+#' tse <- cluster(tse, 
+#'                MARGIN="samples", 
```

Comment:
brings to my mind: do we also accept "rows" and "cols" as MARGIN? I think it would make sense. That should then apply to all functions that use MARGIN.

Reply:
I agree that it's better for the user. But for the code it isn't great to have multiple values that have the same purpose. For now I've put 'rows', 'features', 'cols', and 'samples' as accepted values but I think we should maybe discuss this further.

Reply:
In transformCounts, samples and features are the "official" values (used in documentation and in error message). --> 1, 2, rows, cols are also working, though